controllers/apply/awards-for-all/fields.js

```diff
@@ -829,6 +910,13 @@ module.exports = function fieldsFor({ locale, data = {} }) {
                         en: 'Enter your organisation’s Companies House number',
                         cy: ''
                     })
+                },
+                {
+                    type: 'string.max',
+                    message: localise({
+                        en: `Organisation’s Companies House number must be ${FREE_TEXT_MAXLENGTH.large} characters or less`,
```

Comment:
`Companies House number must be ${FREE_TEXT_MAXLENGTH.large} characters or less`